src/realTimeReporter/reporter.ts

```diff
@@ -45,8 +46,26 @@ export default class Reporter {
     return getLastItem(this.testItems);
   };
 
+  private getTestItemByName(itemName: string, eject: boolean = false): TestItem {
+    if (itemName) {
+      const itemIndex = this.testItems.findIndex((item) => item.name === itemName);
+
+      if (itemIndex !== -1) {
+        const item = this.testItems[itemIndex];
+
+        if (eject) {
+          this.testItems.splice(itemIndex, 1);
```

Comment:
Please, avoid side-effect in `get...`-function. Just return a corresponding value.

Reply:
Fixed